extensions/amp-story/1.0/amp-story.js

```diff
@@ -2694,11 +2697,29 @@ export class AmpStory extends AMP.BaseElement {
       win.CSS && win.CSS.supports && win.CSS.supports('display', 'grid')
     );
   }
+
+  /**
+   * Downloads Material Icons if bling links are present in the story
+   */
+  getBlingLinkIcons_() {
```

Comment:
Maybe it makes sense to do this in `amp-story-page`'s `setDistance`, so we can only load the fonts when you get close to the page that uses them.  Then you could also just check to see if that link tag is already in the DOM, to prevent bloating the DOM with extra elements, or making extra requests (if the browser didn't cache it somehow).

Distance is how many "pages away" the `amp-story-page` element is from being in view (i.e. the current page has distance 0, the next page has distance 1, the page after that 2, etc.)

Reply:
Thanks for the tip! I'll try it out.

Reply:
So @gmajoulet gave me the idea of using SVGs in the stylesheet, which is nice because we don't have to do any remote calls and we'll only need a handle of icons anyways. One downside is that we can't use pure CSS to resize or recolor the icon, but updating the SVG directly should be easy enough to do with JS.

For this PR I'd like to keep the icon at its default values (black, 24x24) but adding size/color/icon attributes on `amp-story-bling-link` is something I'll work on next.